gcp/apis/compute/v1alpha1/network_types_test.go

```diff
@@ -0,0 +1,17 @@
+/*
+Copyright 2019 The Crossplane Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
```

Comment:
The PR body calls out tests for controller code, but it looks like you are planning on implementing tests here as well. Just wanted to add a note so there is something to check-off before merging since it isn't called out explicitly :)